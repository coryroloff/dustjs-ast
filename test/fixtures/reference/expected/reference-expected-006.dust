{one.two.three.four.five|js|s}