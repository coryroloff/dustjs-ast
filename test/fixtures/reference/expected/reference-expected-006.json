{
	"type": "Template",
	"body": [
		{
			"type": "Reference",
			"id": {
				"type": "Identifier",
				"path": [
					{
						"type": "Key",
						"name": "single-key",
						"loc": {
							"start": {
								"line": 1,
								"column": 2
							},
							"end": {
								"line": 1,
								"column": 12
							}
						}
					}
				],
				"loc": {
					"start": {
						"line": 1,
						"column": 2
					},
					"end": {
						"line": 1,
						"column": 12
					}
				}
			},
			"filters": [
				{
					"type": "Key",
					"name": "js",
					"loc": {
						"start": {
							"line": 1,
							"column": 13
						},
						"end": {
							"line": 1,
							"column": 15
						}
					}
				},
				{
					"type": "Key",
					"name": "s",
					"loc": {
						"start": {
							"line": 1,
							"column": 16
						},
						"end": {
							"line": 1,
							"column": 17
						}
					}
				}
			],
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 18
				}
			}
		}
	],
	"tokens": [
		{
			"type": "Punctuator",
			"value": "{",
			"loc": {
				"start": {
					"line": 1,
					"column": 1
				},
				"end": {
					"line": 1,
					"column": 2
				}
			}
		},
		{
			"type": "Key",
			"value": "single-key",
			"loc": {
				"start": {
					"line": 1,
					"column": 2
				},
				"end": {
					"line": 1,
					"column": 12
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "|",
			"loc": {
				"start": {
					"line": 1,
					"column": 12
				},
				"end": {
					"line": 1,
					"column": 13
				}
			}
		},
		{
			"type": "Key",
			"value": "js",
			"loc": {
				"start": {
					"line": 1,
					"column": 13
				},
				"end": {
					"line": 1,
					"column": 15
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "|",
			"loc": {
				"start": {
					"line": 1,
					"column": 15
				},
				"end": {
					"line": 1,
					"column": 16
				}
			}
		},
		{
			"type": "Key",
			"value": "s",
			"loc": {
				"start": {
					"line": 1,
					"column": 16
				},
				"end": {
					"line": 1,
					"column": 17
				}
			}
		},
		{
			"type": "Punctuator",
			"value": "}",
			"loc": {
				"start": {
					"line": 1,
					"column": 17
				},
				"end": {
					"line": 1,
					"column": 18
				}
			}
		}
	],
	"loc": {
		"start": {
			"line": 1,
			"column": 1
		},
		"end": {
			"line": 1,
			"column": 18
		}
	}
}